tests/tests.fmf

```diff
@@ -15,6 +15,8 @@ duration: 2h
     # prterun blocks running mpiexec as root. Set some environment variables to workaround it
     OMPI_ALLOW_RUN_AS_ROOT: 1
     OMPI_ALLOW_RUN_AS_ROOT_CONFIRM: 1
+  # TODO: Investigate why openmpi tests are significantly slower
+  duration: 6h
```

Comment:
Can anyone upstream help investigate this? This is an issue I've noticed very early on in the packaging, also in [`dbcsr`](https://kojipkgs.fedoraproject.org//packages/dbcsr/2.6.0/2.fc40/data/logs/x86_64/build.log)

Reply:
With openmpi (by contrast to mpich) the `mpiexec` command might need the flag `--bind-to none` for good performance (e.g. like [here](https://github.com/cp2k/cp2k-containers/blob/master/docker/2024.1_openmpi_generic_psmp.Dockerfile#L95) for regression testing with openmpi using docker containers).

Reply:
Do you know of an environment variable to set these? It could be useful to suggest these in the upstream guidelines or default macros as well.

Reply:
Maybe setting `OMPI_MCA_hwloc_base_binding_policy` to `none` has the same effect.

Reply:
I actually made `--bind-to none` the default in #3191.
For running a sub-sets of tests there is already `--restrictdirs` and `--skipdir`, which both accept regular expressions. However, I'd suggest to stick to `--smoketest`. If that takes too long then I think it's more honest to not run any tests at all.

Reply:
Testing wise, running the full regression tests is not bad, with only 4 cores, the tests took ~30 min for both serial and mpich. It's just OpenMPI that has this slow down. I can also ask upstream how much they are willing to offer to eliminate the testing queue quicker. It needs to be experimented a bit